### PR TITLE
Uml 98 show valid lpa details

### PR DESCRIPTION
**Pull request by taylorsj1980**

### CODE REVIEW

File: service-front/app/src/Viewer/templates/app/view-lpa.html.twig [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{% extends '@layout/default.html.twig' %}
+
+{% block title %}View LPA{% endblock %}
+
+{% block content %}
+    <style>
```
Comment (cooperaj) [inline service-front/app/src/Viewer/templates/app/view-lpa.html.twig]: I'm guessing the inline styles are temporary?

File: service-front/app/src/Viewer/templates/app/view-lpa.html.twig [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+            </div>
+
+            <div class="govuk-grid-row">
+
+                <div class="govuk-grid-column-full">
+
+                    <div class="govuk-panel govuk-panel--confirmation">
+
+                        <h1 class="govuk-panel__title">This LPA is {{  lpa.isValid ? 'valid' : 'invalid' }}</h1>
```
Comment (cooperaj) [inline service-front/app/src/Viewer/templates/app/view-lpa.html.twig]: Extra space "{{  lpa.isValid..."

File: service-front/app/test/ViewerTest/Handler/EnterCodeHandlerTest.php [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                ->willReturn($returnData);
+        }
+
+        $symfonyFormProphecy->createView()
+            ->willReturn($formProphecy->reveal());
+
+        $formFactoryProphecy = $this->prophesize(FormFactoryInterface::class);
+
+        $formFactoryProphecy->create(ShareCode::class, null, [
+                'csrf_token_manager' => self::CSRF_TOKEN,
```
Comment (cooperaj) [inline service-front/app/test/ViewerTest/Handler/EnterCodeHandlerTest.php]: The token manager is an object retrieved from the request attributes - an instance of TokenManager.

File: service-front/app/src/Viewer/templates/app/view-lpa.html.twig (hunk 1 above)
Comment (taylorsj1980) [reply service-front/app/src/Viewer/templates/app/view-lpa.html.twig]: Yes definitely. I wanted to match the prototype exactly as requested in this ticket so needed to create custom styles but I think there might be some rework to the prototype to make it adopt GDS styles. I'll pick this up with Katie and Darren.

File: service-front/app/src/Viewer/templates/app/view-lpa.html.twig (hunk 2 above)
Comment (taylorsj1980) [reply service-front/app/src/Viewer/templates/app/view-lpa.html.twig]: Will be fixed.

File: service-front/app/test/ViewerTest/Handler/EnterCodeHandlerTest.php (hunk 3 above)
Comment (taylorsj1980) [reply service-front/app/test/ViewerTest/Handler/EnterCodeHandlerTest.php]: Good spot. Will change.

**Pull request merged**